README.md

```diff
@@ -262,23 +262,13 @@ We provide multi-GPU training examples with distributed gather and synchronized
 ## Benchmarks
 
 Implemented models and their performance on various datasets. Hyperparameters are not
-tuned for maximum accuracy. For detailed results and more info about the benchmarks click
+tuned for maximum accuracy. For detailed results and more information about the benchmarks click
 [here](https://docs.lightly.ai/self-supervised-learning/getting_started/benchmarks.html).
 
 
-### Imagenet
+### ImageNet1k
 
-The following experiments have been conducted on a system with 2x4090 GPUs. 
-Training a model takes around 4 days for 100 epochs (35 min per epoch), including kNN, linear probing, and fine-tuning evaluation.
-
-> **Note**: Evaluation settings are based on these papers:
-> * Linear: [SimCLR](https://arxiv.org/abs/2002.05709)
-> * Finetune: [SimCLR](https://arxiv.org/abs/2002.05709)
-> * KNN: [InstDisc](https://arxiv.org/abs/1805.01978)
-> 
-> See the [benchmarking scripts](./benchmarks/imagenet/resnet50/) for details.
```

Comment:
I would keep this part to avoid confusion as the results are different than in the original papers.